docs/pages/distribution/turtle-cli.md

```diff
@@ -44,23 +45,26 @@ $ npm install -g turtle-cli
 
 Then run `turtle setup:ios` and/or `turtle setup:android` to verify everything
 is installed correctly. This step is optional and is also performed during
-the first run of the Turtle CLI. Please note that the Android setup command
+the first run of `turtle build:[ios|android]`. Please note that the Android setup command
 downloads, installs, and configures the appropriate versions of the Android SDK
 and NDK.
 
 If you would like to make the first build even faster, you can supply the Expo
-SDK version to the setup command like so: `turtle setup:ios --sdk-version 30.0.0`.
+SDK version to the setup command like so: `turtle setup:ios --sdk-version 38.0.0`.
 This tells Turtle CLI to download additional Expo-related dependencies for
 the given SDK version.
 
 All Expo-related dependencies will be installed in a directory named `.turtle`
 within your home directory. This directory may be removed safely if you ever
 need to free up some disk space.
 
-## Publish your project
+## Publish your project!
+
+When you're building standalone apps with Turtle CLI, the build process is happening on your local machine.
+Turtle CLI makes use of exactly the same codebase which is running on our servers (`expo build:[ios|android]` command).
+This means you're required to publish your app to Expo servers or host it on your own server prior to building it with Turtle CLI.
```

Comment:
```suggestion
This means you're required to publish your app to Expo's servers or host it on your own server **prior to building it with Turtle CLI**. Whether you want Expo to host your app, or you'd like to do it yourself, all you need to do is follow the appropriate guide:

- [Publishing an app to Expo's servers](../../workflow/publishing/)
- [Hosting an app on your own server](../hosting-your-app/)
```

Reply:
bolded because people tend to miss that part, and also put the guide links in bullets so they're easy to see when skimming